src/test/java/SetTest.java

```diff
@@ -0,0 +1,58 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class SetTest {
+
+        private Set<Integer> numbers;
+
+        @BeforeEach
+        void setUp() {
+                numbers = new HashSet<>();
+                numbers.add(1);
+                numbers.add(1);
+                numbers.add(2);
+                numbers.add(3);
+        }
+
+        @Test
+        public void Set_크기_확인() {
+
+                // given
+                // when
+                // then
+                assertEquals(3, numbers.size());
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

assertj의 assertThat 사용 추천
collection의 경우 위와 같이 구현도 가능함

Reply:
오 assertj 의 assertThat 에서 제공해주는 함수들이 굉장히 직관적이라 가독성이 훨씬 좋아질 것 같습니다!
꿀팁 감사합니다.